osmnx/utils_graph.py

```diff
@@ -5,7 +5,6 @@
 
 import geopandas as gpd
 import networkx as nx
-import numpy as np
```

Comment:
I removed this line from your branch to satisfy the linter.

Reply:
Sorry, leftover from numpy-based creation of an empty array that I later replaced with that list.